app/javascript/react/screens/App/common/ListViewToolbar/ListViewToolbar.js

```diff
@@ -288,6 +300,7 @@ class ListViewToolbar extends Component {
   };
 
   render() {
+    const filteredListItems = this.filterListItems();
```

Comment:
Can we move the return value of this function call to the argument object below :)

Reply:
The reason I didn't move this to the argument object is because I wanted to both pass it directly into the object, and also pass it into `this.filterSortPaginateListItems(filteredListItems)`, so we don't have to run the filtering twice per render. I think instead I should actually remove the one we use as a default argument for `filterSortPaginateListItems`, as that is no longer exposed.

Reply:
Ah, oops, sorry I totally missed that :P